packages/generator/src/edmx-to-vdm/v2/function-import.ts

```diff
@@ -17,28 +18,36 @@ export function generateFunctionImportsV2(
 ): VdmFunctionImport[] {
   const edmxFunctionImports = parseFunctionImports(serviceMetadata.edmx.root);
 
-  return edmxFunctionImports.map(f => {
-    const httpMethod = f['m:HttpMethod'].toLowerCase();
-    const swaggerDefinition = swaggerDefinitionForFunctionImport(
-      f.Name,
-      httpMethod,
-      serviceMetadata.swagger
-    );
-    return {
-      ...transformFunctionImportBase(
-        f,
-        f.Parameter,
-        swaggerDefinition,
-        formatter
-      ),
-      httpMethod,
-      returnType: parseFunctionImportReturnTypes(
-        f.ReturnType,
-        entities,
-        complexTypes,
-        extractResponse(f.Name),
-        serviceMetadata.edmx.oDataVersion
-      )
-    };
-  });
+  return edmxFunctionImports
+    .map(f => {
+      const httpMethod = f['m:HttpMethod'].toLowerCase();
+      const swaggerDefinition = swaggerDefinitionForFunctionImport(
+        f.Name,
+        httpMethod,
+        serviceMetadata.swagger
+      );
+
+      // TODO 1571
+      if (f.Parameter && hasUnsupportedParameterTypes(f.Parameter)) {
+        return undefined;
+      }
```

Comment:
I think we should just filter the functionImports before the map by doing this, i. e.
```ts
return edmxFunctionImports
  .filter(f => f.Parameter && hasUnsupportedParameterTypes(f.Parameter))
  .map(f => ...
```